pom.xml

```diff
@@ -3,7 +3,7 @@
     <name>Sakai Maven Plugin</name>
     <groupId>org.sakaiproject.maven</groupId>
     <artifactId>plugins</artifactId>
-    <version>1.4.6-SNAPSHOT</version>
+    <version>1.4.7-SNAPSHOT</version>
```

Comment:
I don't think we ever actually released 1.4.6, the version bump should only be happening via the maven release plugin. This process is documented in the README.